### ISSUE TITLE
ioredis isn't supported

### ISSUE THREAD

**Issue by unity** [facundoolano/redis-lru#9]
`ioredis` isn't supported, Tried it in a project, didn't work. Then saw tests fail.

```
yarn add ioredis
````

Then, in `/test/index.js`

```
const redis = require('ioredis').createClient(6379, 'localhost');
```

Shows many tests failing. I've forked it, will try to see how big the work is to add support. Spent 15 minutes, it's not going to be a 1 line fix.

**Comment by facundoolano** [comment]
I did test using IORedis back when I implemented this, I wonder if something has changed. Does it work if you use the regular redis client?

**Comment by dearpeach** [comment]
how to use ioredis to support sentinel?
it doesn't work? does it have some example?

**Comment by facundoolano** [comment]
should be fixed by #14
